io.github.dman95.SASM.yml

```diff
@@ -0,0 +1,36 @@
+---
+
+# SPDX-License-Identifier: MIT OR CC0-1.0
+# SPDX-FileCopyrightText: 2021 dgcampea <59268455+dgcampea@users.noreply.github.com>
+app-id: io.github.dman95.SASM
+
+runtime: org.kde.Sdk
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+
```

Comment:
you probably need a `rename-destkop` and `rename-icon`

Also there is no appstream file. It's required for flathub

Reply:
that would have been nice to separate XDG stuff from the rest. The path is big with very little relevant to the flatpak that I didn't even check what was in there.

Reply:
It's not really irrelevant to flatpak, the patch/PR was created specifically to easily package this as a flatpak, otherwise it would have required a bunch of shell commands to accomplish it.

Reply:
at first sight it didn't look like it. Anyway. Still.